src/data-model-providers/codegen/CodegenDataModelProvider_Write.cpp

```diff
@@ -198,6 +198,28 @@ DataModel::ActionReturnStatus CodegenDataModelProvider::WriteAttribute(const Dat
     return CHIP_NO_ERROR;
 }
 
+void CodegenDataModelProvider::ListAttributeWriteNotification(const ConcreteAttributePath & aPath,
+                                                              BitFlags<DataModel::ListWriteOperation> opType)
+{
+    AttributeAccessInterface * aai = AttributeAccessInterfaceRegistry::Instance().Get(aPath.mEndpointId, aPath.mClusterId);
+
+    if (aai != nullptr)
+    {
+        switch (opType)
+        {
+        case DataModel::ListWriteOperation::kListWriteBegin:
```

Comment:
This doesn't make sense; `opType` is not of type `DataModel::ListWriteOperation`.

Reply:
Yes, this can be confusing and wasn't my intention. I'm removing the BitFlags from the parameter to make clear that it is a `DataModel::ListWriteOperation` and because `BitFlags` is not needed.

It seems to work right now thanks to the `operator FlagsEnum` overload inside the BitFlags class.